tokio-trace-serde/Cargo.toml

```
@@ -0,0 +1,9 @@
+[package]
+name = "tokio-trace-serde"
+version = "0.1.0"
+authors = ["J Haigh <debugsteven@gmail.com>"]
+edition = "2018"
+
```

We should probably also add:
```suggestion
license = "MIT"
```